tests/test_event_base.py

```diff
@@ -33,8 +33,6 @@ def test_invalid_ip_should_raise_exception(base_events):
         base_events(1, ip="invalid_ip")
     with pytest.raises(ValidationError):
         base_events(1, ip="  ")
-    with pytest.raises(ValidationError):
-        base_events(1, ip="051.255.9.27")
```

Comment:
As the regex ip validation was removed this test does not raise an exception and silently corrects the ip from "051.255.9.27" to "51.255.9.27".
This causes the "event" to change after deserialisation/serialisation.
If this is not a problem it's good for me - removing this test

Reply:
It's not a problem to me.